plugins/link/dialogs/anchor.js

```diff
@@ -136,10 +136,20 @@ CKEDITOR.dialog.add( 'anchor', function( editor ) {
 				label: editor.lang.link.anchor.name,
 				required: true,
 				validate: function() {
-					if ( !this.getValue() ) {
+					var disallowedWhitespacesRegex = /[\u0020\u0009\u000c]/g,
```

Comment:
I would add also new line and carriage return characters here – just to be sure. They aren't possible to but they can be pasted from somewhere. Better safe than sorry.

It'd be also nice to link to the spec to explain why these characters are not allowed.

Reply:
I've added a `new line` and `carriage return` character but I was not able to add a unit test to it as it was impossible to paste it to input. Are we okay with it? 🤔 

I've also added also short information about disallowed characters from spec.